program/rust/src/lib.rs

```diff
@@ -29,18 +26,18 @@ use crate::log::{
 use processor::process_instruction;
 
 use solana_program::entrypoint::deserialize;
-use solana_program::program_error::ProgramError;
 use solana_program::{
     custom_heap_default,
     custom_panic_default,
 };
 
 //Below is a high lever description of the rust/c setup.
 
-//As we migrate from C to Rust, our Rust code needs to be able to interract with C
-//build-bpf.sh is set up to compile the C code into a bpf archive file, that build.rs
-//is set up to link the rust targets to. This enables to interact with the c_entrypoint
-//as well as similarly declare other C functions in Rust and call them
+//As we migrate from C to Rust, our Rust code needs to be able to interact with C
+//build-bpf.sh is set up to compile the C code into a two archive files
+//contained in `./program/c/target/`
+// - `libcpyth.a` contains the bpf version for production code
+// - `libcpythtest.a` contains the systems architecture version for tests
```

Comment:
fwiw would be nice if the names of these files corresponded to the architectures they were built for, like `libcpyth-bpf` and `libcpyth-native`